companion/src/simulation/telemetrysimu.cpp

```diff
@@ -154,6 +156,55 @@ void TelemetrySimulator::onLogTimerEvent()
   logPlayback->stepForward(false);
 }
 
+void TelemetrySimulator::onGpsRunLoop()
+{
+    int a = ui -> gps_latlon -> text().contains(",");
+    if(not a){
+        QMessageBox::information(this,tr("Bad GPS Format"),tr("Must be decimal lat,lon"));
```

Comment:
Where practical do not use abbreviations as the auto translation will not be possible which means the translator devs have to perform manually as follow up PRs or may generate unexpected results.

Reply:
The only abbreviations are "lat,lon". Should I spell it out as latitude, longitude?

Reply:
Yes. Since this is a message there is no ui space limitation. Sometimes have to use abbreviations in ui dialogs but try to avoid.